registry/tests/admin_test.py

```diff
@@ -228,3 +227,122 @@ def testPasswordReset(self):
             }
         )
         assert resp.status_code == requests.codes.ok
+
+    def testCreateUser(self):
+        QUILT_AUTH_URL = quilt_server.app.config['QUILT_AUTH_URL']
```

Comment:
I'd just do `from quilt_server.views import QUILT_AUTH_URL` at the top of the file - unless that breaks something?

(If it works, fix the existing tests, too.)